### PR TITLE
feat: add unit formatting to pie chart

### PR DESCRIPTION
**Pull request by pauldambra**
## Problem

The pie chart doesn't yet have the unit formatter

## Changes

* Adds unit formatter to pie chart
* obeys the linter to remove some custom styling

## How did you test this code?

running it locally and seeing it work

### CODE REVIEW

**Review by Twixes — APPROVED** [review]
All good with formatting, though the pie is off center… but that's outside the scope for now.

File: frontend/src/scenes/insights/aggregationAxisFormat.ts [hunk 1]
```diff
@@ -53,6 +53,7 @@ export const canFormatAxis = (chartDisplayType: ChartDisplayType | undefined): b
             ChartDisplayType.ActionsBarValue,
             ChartDisplayType.ActionsTable,
             ChartDisplayType.WorldMap,
+            ChartDisplayType.ActionsPie,
```
Comment (Twixes) [inline frontend/src/scenes/insights/aggregationAxisFormat.ts:56]: BTW looking at the UI now, I think that the axis _can_ be formatted everywhere where the "Unit" selector is shown – so do we still need `canFormatAxis()`?

Comment (pauldambra) [reply frontend/src/scenes/insights/aggregationAxisFormat.ts:56]: Ah ha! Yes! This is the final (brace yourself) piece of the pie. Good spot 🥇

Comment (pauldambra) [reply frontend/src/scenes/insights/aggregationAxisFormat.ts:56]: 🤔 Does it make sense on stickiness ?

Comment (pauldambra) [reply frontend/src/scenes/insights/aggregationAxisFormat.ts:56]: No... I'll follow-up so it's only show if type is trends

Comment (pauldambra) [reply frontend/src/scenes/insights/aggregationAxisFormat.ts:56]: https://github.com/PostHog/posthog/pull/11143

**Pull request merged**